frontend/catalyst/pennylane_extensions.py

```diff
@@ -295,15 +305,15 @@ def grad(f: DifferentiableLike, *, method=None, h=None, argnum=None):
     Args:
         f (Callable): a function or a function object to differentiate
         method (str): The method used for differentiation, which can be any of
-                      ``["fd", "ps", "adj"]``,
+                      ``["fd", "mixed"]``,
             where:
 
-            - ``"fd"`` represents first-order finite-differences,
-
-            - ``"ps"`` represents the two-term parameter-shift rule, supported by the Pauli
-              rotation gates,
+            - ``"fd"`` represents first-order finite-differences for the entire hybrid
+              circuit,
 
-            - ``"adj"`` represents the adjoint differentiation method.
+            - ``"mixed"`` represents deferring the quantum differentiation to the method
+              specified by the QNode, while the classical computation is differentiated
+              using finite differences.
```

Comment:
Might want to pre-emptively change this since I have a feeling we'll forget about it in the Enzyme PRs.
```suggestion
              using traditional auto-diff.
```